stock-tracker/src/features/search/redux/searchReducer.ts

```diff
@@ -14,7 +14,7 @@ export interface SearchState {
   error: boolean;
 }
 
-const initialState = {
+export const initialState = {
```

Comment:
I don't think that this needs to be exported.

Reply:
Thanks, I added it while trying something out and forgot to delete!